src/app/learn/get-started/advanced/route-params/index.html

```diff
@@ -4,26 +4,29 @@
   navTitle="Route parameters">
 
   <p>
-    In addition to automatically registering routes and creating components for all <stache-code>index.html</stache-code> files in the <stache-code>src/app</stache-code> directory, SKY UX supports routes that contain <a href="https://angular.io/guide/router#route-definition-with-a-parameter">route parameters</a>.
+    SKY UX supports routes that contain <a href="https://angular.io/guide/router#route-definition-with-a-parameter">route parameters</a>. This complements the fact that SKY UX Builder automatically registers routes and creates components for all <stache-code>index.html</stache-code> files in the <stache-code>src/app</stache-code> directory.
   </p>
   <p>
     To create a route parameter, you create a subfolder and prefix the name with an underscore. The subfolder name must be a valid TypeScript variable because SKY UX Builder creates a variable with that name (minus the underscore). The type of the variable is a string. For example, to use a route parameter in a <stache-code>src/app/dogs</stache-code> folder, you can create a <stache-code>src/app/dogs/_id</stache-code> subfolder and then create an <stache-code>index.html</stache-code> file with the following content:
   </p>
-
   <stache-code-block>
     You are viewing dog {{ id }}.
   </stache-code-block>
-
   <p>
     When users visit the route <stache-code>dogs/42</stache-code>, the <stache-code>id</stache-code> variable is set to "42" and renders as:
   </p>
-
   <stache-code-block>
     You are viewing dog 42.
   </stache-code-block>
+  <p>
+    We recommend against creating paramaterized routes as siblings of static routes. Nesting paramaterized routes alongside static routes is not a problem, but when paramaterized routes and static routes are siblings, it can create ambiguity for the router. 
```

Comment:
I'm wondering if we could beef up this sentence (possibly with a couple small examples) to also address https://github.com/blackbaud/skyux2-docs/issues/385?

Reply:
Good call. I'll work on this.